dracut/30ignition/ignition-kargs-helper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Send out a warning message to contact the distribution and exit.
+# Distributions implementing Ignition kargs can either disable the unit
+# or override this binary with the actual implementation.
+echo "Ignition kargs requires an additional binary. Please contract your distribution."
```

Comment:
s/contract/contact/.  Maybe something like `Ignition kernelArguments is not supported by this Linux distribution.\nAsk your distribution to implement ignition-kargs-helper.`?